sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -9,24 +9,17 @@
       Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.
       Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Compute" (https://www.nuget.org/packages/Azure.ResourceManager.Compute/). Please see the package changelog for more information.
     </Description>
-    <Version>44.0.0.0</Version>
+    <Version>45.0.0.0</Version>
```

Comment:
Make sure to also update the file sdk/compute/Microsoft.Azure.Management.Compute/AzSdk.RP.props, similar to https://github.com/Azure/azure-sdk-for-net/pull/18498/files#diff-3d5895728d5573d745e564d34e730349416ca38f5866877e43201c05b25c3d63L4

Reply:
the file already has the change.